aws-athena/views/reporting-vw_pin_township_class.sql

```diff
@@ -20,30 +20,8 @@ SELECT
     leg.user1 AS township_code,
     SUBSTR(correct.nbhd, 3, 3) AS nbhd,
     correct.class,
-    CASE
-        WHEN SUBSTR(correct.class, 1, 2) IN ('EX', 'RR') THEN correct.class
-        -- OA classes contain their major class as the third and final
-        -- character
-        WHEN SUBSTR(correct.class, 1, 2) = 'OA' THEN SUBSTR(correct.class, 3, 1)
-        WHEN correct.class IN (
-                '500', '535', '501', '516', '517', '522', '523',
-                '526', '527', '528', '529', '530', '531', '532',
-                '533', '535', '590', '591', '592', '597', '599'
-            ) THEN '5A'
-        WHEN correct.class IN (
-                '550', '580', '581', '583', '587', '589', '593'
-            ) THEN '5B'
-        ELSE SUBSTR(correct.class, 1, 1)
-    END AS major_class,
-    CASE WHEN correct.class IN ('299', '399') THEN 'CONDO'
-        WHEN correct.class IN ('211', '212') THEN 'MF'
-        WHEN
-            correct.class IN (
-                '202', '203', '204', '205', '206', '207',
-                '208', '209', '210', '234', '278', '295'
-            )
-            THEN 'SF'
-    END AS property_group,
+    groups.reporting_class_code AS major_class,
+    groups.modeling_group AS property_group,
```

Comment:
**praise:** Nice refactor, we love <3

Reply:
I think it's close to 100% of the diff, besides classes like 999 and 0. Yes, https://github.com/ccao-data/data-architecture/issues/362 will reduce it, to what degree I'm not sure.